admin_guide/manage_users.adoc

```diff
@@ -66,6 +115,40 @@ authentication provider after starting to use {product-title}, the user names
 that overlap will not work because of the entries in the identity list, which
 will still point to the old authentication method.
 
+[[admin-guide-managing-users-creating-groups]]
+== Creating Groups
+
+While a user is an entity making requests to {product-title}, users can be
+organized into one or more groups made up from a set of users. Groups are useful
+for managing many users at one time, such as for authorization policies, or to
+grant permissions to multiple users at once.
```

Comment:
We should link to the ldap docs on syncing groups here, to outline that that process is done automatically and that this is only done for the other providers where this is not done automatically.